index.html

```diff
@@ -1,232 +1,255 @@
+
+      <a class="social-media">LET'S CONNECT</a>
+      
+      <ul class="social-icons">
+         <li><a href="#"><img src="image/Vector2.png" alt=""></a></li>
+          <li><a href="#"><img src="image/Linkedin icon (1).png" alt=""></a></li>
+          <li><a href="#"><img src="image/Vector3.png" alt=""> </a></li>
+          <li><a href="#"><img src="image/Vector5.png" alt=""> </li>
+          
+            <li><a href="#"><img src="image/Vector1.png" alt=""></a></li>
+        </ul>
 
 
 
+      
+     
+    <button type="button" class="btn-in btn-anima">Get my resume</button>
+    </div>
+    <ul class="user-tool">
+      <ul class="languages">
+        <div class="user-tool-d">
+          <h3>Languages</h3>
+          <img src="image/IconDown.png" alt="arrow-down" />
+        </div>
+        <ul class="tool-list">
+          <li>
+            <img src="image/JS.png" alt="javascript" />
+            <p>Javascript</p>
+          </li>
+          <li>
+            <img src="image/Ellipse11.png" alt="html" />
+            <p>HTML</p>
+          </li>
+          <li>
+            <img src="image/Ellipse12.png" alt="css" />
+            <p>CSS</p>
+          </li>
+        </ul>
+      </ul>
+      <div class="frameworks">
+        <div class="user-tool-d">
+          <h3>Frameworks</h3>
+          <img src="image/drop-down.png" alt="arrow bottom icon" />
+          </div>
+        <hr/>
+      </div>
+      <div class="skills">
+        <div class="user-tool-d">
+          <h3>Skills</h3>
+          <img src="image/drop-down.png" alt="arrow bottom icon" />
+        </div>
+      <hr>
+        
+      </div>
+    </ul>
+  </section>
+  
+  <section class="contact-form">
+    <div class="contact-form-holder">
+      <h2 class="contact-title">Contact me</h2>
+
+      <p class="p-pargraph">
+        If you have an application you <br> are interested in  developing, a <br> feature  
+        that you need built or a <br> project that needs coding. I’d <br> love to help
+        with it
+      </p>
+
+      <form
+        action="https://formspree.io/f/mzbowdel" class="responsive-form" method="post">
+        <ul class="ul-form">
+          <li class="li-form">
+            <input type="text" placeholder="Tshepo" name="name" id="name_form" maxlength="30" required/>
+          </li>
+          <li class="li-form">
+            <input type="email" placeholder="tshepomakama45@gmail.com" name="email" id="email_form" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required/>
+          </li>
+          <li class="li-form">
+            <textarea name="message" placeholder="Write your message here" id="message" cols="30" rows="10" maxlength="500" required></textarea>
+          </li>
+         </ul>
+         <div>
+          <input id="get-submit"  type="submit" value="Get It Touch" class="btn-n"/>
+          <img src="image/Separator2.png" class="shape"  alt="shape">
+        </div>
+         
+        
+      </form>
+    </div>
+  </section>
```

Comment:
Well done on this project.

I notice a few things that I believe will improve your project even better. Ple note as follows 👇🏾 
- There should be a background illustration in this section.
- The space between the input elements is too large and should follow the size as seen in the Figma template.
- The `get in touch` button does not match the button in the Figma template.
- The paragraph text below the `contact me` title is not well aligned in comparison with the Figma design.


Your projects | Figma Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/221877209-78d3ba9e-1113-4f88-812f-941ad6f09335.png) | ![image](https://user-images.githubusercontent.com/84156714/221877789-0646cec7-420f-4113-b95b-b7f83f705ef6.png) 
 
 ![image](https://user-images.githubusercontent.com/84156714/221878128-46222c90-139b-4c9c-b03f-4daeffd57a81.png)

Reply:
Please treat the above review comments as changes required as it was mistakenly selected as **approved**.